src/System.Runtime.Extensions/tests/System/Environment.SetEnvironmentVariable.cs

```diff
@@ -61,6 +61,7 @@ public void IncorrectVariableThrowsArgument()
         }
 
         [Theory]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap | TargetFrameworkMonikers.UapAot, "Environment Variables monidification isn't supported in Appx.")]
```

Comment:
conversely, when it IS appropriate to permanently disable a tes tor part of it on uap/uapaot, we should have a comment explaining why -- inline like this for easy grepping if possible
`[SkipOnTargetFramework(TargetFrameworkMonikers.Uap)] // because frob is not supported on uap`
This is because later we will have to do a pass through them all to catch any cases where one was disabled that actually should have worked.
I'll remind the larger team at the standup.